### PR TITLE
Add style to your html code

### PR DESCRIPTION
**Pull request by obamgbopa**

### CODE REVIEW

**Review by theoomoregbee — CHANGES_REQUESTED** [review]
as your background color #f8fdff and make it be the body background instead.
The fonts also is not the right one (font-family: "Raleway", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;).
The logo is not properly sized (and no devCenter just the DC)
Observed you copied bootstrap codes 😉

File: frontend/explainer.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div class="container">
         <div class="row">
             <div class="nav-icon col-6">
-                <a href="#"><img src="assets/img/devcenter-logo.svg"> </a>
+                <a href="#"><img src="assets/img/logo.png"> </a>
```
Comment (theoomoregbee) [inline frontend/explainer.html]: Use width:64px; height:64px for the image

**Review by theoomoregbee — COMMENTED** [review]
reduce the width of your container 

and also make your link buttons to continue e.g How to have this background color:`#32ad75` and the padding to be     `padding: 10px 30px;` , then curvy borders with `border-radius: 2px;`.

File: frontend/explainer.html [hunk 2]
```diff
@@ -2,29 +2,47 @@
...
-        <h3>All you have to do is try to answer the next couple of question as best as you can so we can figure out what you may like.</h3>
+        <div class="row">
+            <div class="col-12" style="text-align:center">
+                <h3>All you have to do is try to answer the next couple of question as best as you can so we can figure out what you may like.</h3>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-12" style="text-align:center">
+                <h3>We then use that to create a track that might be the most engaging way for you to get into tech.</h3>
```
Comment (theoomoregbee) [inline frontend/explainer.html]: use color `#777` for all your secondary text after the main header
with this  `font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;`

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

File: frontend/assets/css/style.css [hunk 3]
```diff
@@ -126,19 +167,28 @@ body {
...
 #inner a:hover,
 #inner a:active {
     background-color: #5FBA87;
 }
 
+h2,
+h3 {
+    color: #777;
+    font-family: "apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "sans-serif";
+}
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css]: add this to it ->  font-weight: 500; to reduce the boldness

**Review by theoomoregbee — CHANGES_REQUESTED** [review]
Only secondary texts should the color i sent initially, why the Primary should still remain black

add link to the `dc` logo to always go back to index.html

File: frontend/category.html [hunk 4]
```diff
@@ -2,44 +2,52 @@
...
-        <div class="background">
-            <div class="transbox">
-                <p>Secondary School Graduate</p>
+    <div class="container">
+        <div class="row">
+            <div class="nav-icon col-6">
+                <a href="#"><img src="assets/img/logo.png"> </a>
+            </div>
+            <div class="col-6 nav-link" style="text-align:right">
+                <h1>Coding101</h1>
```
Comment (theoomoregbee) [inline frontend/category.html]: use h3 instead

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

File: frontend/assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    -ms-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #f8fdff;
+    font-family: "Raleway", "HelveticaNeue", "Helvetica Neue", "Helvetica", "Arial", "sans-serif";
+}
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css]: add `letter-spacing: 0.3px;` to make each letter ve small spacing in between

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

File: frontend/assets/css/style.css (hunk 5 above)
Comment (theoomoregbee) [reply frontend/assets/css/style.css]: add` letter-spacing: 1.2px;` for the button links

Comment (obamgbopa) [reply frontend/assets/css/style.css]: Done

**Review by theoomoregbee — APPROVED** [review]

File: frontend/assets/css/style.css [hunk 6]
```diff
@@ -144,6 +152,7 @@ body {
     text-align: center;
     padding: 15px;
     padding-top: 80px;
+    border-radius: 10px;
 }
 
 .transbox p {
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css]: For this, make the font size to be `15px` , then remove the font-weight, then letter spacing of `1px`

File: frontend/assets/css/style.css [hunk 7]
```diff
@@ -130,10 +130,18 @@ body {
     */
 }
 
+div .background {
+    -webkit-box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
+    -moz-box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css]: the shadow is too much `0.75` try `0.4`, then 
3px`(x axis)` 4px`(y axis)` 5px`(reduce the blur)` -2px`(reduce the shadow spreading)`

to make the shadow, be on bottom and right based on `x and y axis`

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

**Review by theoomoregbee — CHANGES_REQUESTED** [review]
This particular commit distorted the other pages, check all pages

File: frontend/assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+    width: 100%;
+    opacity: 0.8;
+    filter: alpha(opacity=60);
+    text-align: center;
+    padding: 15px;
+    padding-top: 25px;
+    border-radius: 10px;
+}
+
+.transbox p {
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css:173]: add padding top make the text be at the middle of the `transbox`

File: frontend/assets/css/style.css [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: #ccc;
+    border-radius: 3px;
+}
+
+.questions-bx,
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css]: make the box look like a card, add box shadow to it, and padding (the content are too close to the borders)

File: frontend/assets/css/style.css [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+    /* background-color: rgba(80, 210, 104, 0.98);
+    padding: 14px 25px; */
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    background-color: #66A1E8;
+    padding: 10px 30px;
+    border-radius: 2px;
+    letter-spacing: 1.2px;
```
Comment (theoomoregbee) [inline frontend/assets/css/style.css:241]: add margin left or right to space two buttons when they are close to each other

Comment (theoomoregbee) [reply frontend/assets/css/style.css:241]: for the `#inner` remove the `width` so any text we place inside won't break

File: frontend/question.html [hunk 11]
```diff
@@ -3,64 +3,100 @@
...
-                </li>
-
+                </div>
             </div>
         </div>
 
-        <div>
-            <div id="button">
-                <button type="button">Back</button>
+        <div class="row">
```
Comment (theoomoregbee) [inline frontend/question.html]: so no need for this row here, just use (after making the reviewed changes on `styles.css`)
```
 <p id="inner" align="center">
   <a href="#">Back</a>
   <a href="#">Next</a>
</p>
```

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

File: frontend/question.html (hunk 11 above)
Comment (obamgbopa) [reply frontend/question.html]: Done

**Pull request merged**